Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -949,7 +949,7 @@ def add_custom_ip_feeds(client: PrismaCloudComputeClient, args: dict) -> Command
         CommandResults: command-results object.
     """
     # the api overrides the blacklisted IPs, therefore it is necessary to add those who exist to the 'PUT' request.
-    current_ip_feeds = client.get_custom_ip_feeds().get("feed", [])
+    current_ip_feeds = (client.get_custom_ip_feeds() or []).get("feed") or []
```

Comment:
It is failed if the function returns None. Please change to:

```suggestion
    current_ip_feeds = (client.get_custom_ip_feeds() or {}).get("feed") or []
```

Reply:
+1, @nericksen this should have been by default a dict and not a list, if you put there a list it would fail again